Create The Things Network doc (#292)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/ttn/_index.md

---
title: "The Things Network"
description: ""
---

{{< figure src="logo_TTN.svg" class="float plain" width="200px" >}}

> The Things Network is a global community building an open-source and decentralized LoRaWAN network.

Reviewer: ```suggestion
The Things Network is a global community building an open, crowdsourced and decentralized LoRaWAN network.
```

> The Things Network runs {{% tts %}}, and is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.

Reviewer: ```suggestion
The Things Network runs {{% tts %}} Community, and is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
```

---
title: "The Things Network"
description: ""
distributions: "The Things Network"
---

> {{< figure src="logo_TTN.svg" class="float plain" width="200px" >}}

Reviewer 2: ```suggestion
{{< figure src="logo_TTN.svg" class="float plain" width="150px" >}}
```

> The Things Network is a global community building an open, crowdsourced, decentralized LoRaWAN network.

Reviewer 2: ```suggestion
The Things Network is a global collaborative Internet of Things ecosystem that creates networks, devices and solutions using LoRaWAN.
```

Reviewer 2: It's important to frame TTN not only as the network, but rather the ecosystem, which also provides and maintains a public network.

Reviewer: Fine with me, but the original sentence has been the initiative description since 2015. I agree it has evolved over time. Let's not block this PR on this though.

Reviewer 2: This sentence is the updated one which we will also use on the new TTN website. cc @wienke @laurensslats

> The Things Network runs {{% tts %}} Community, and is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.

Reviewer 2: ```suggestion
The Things Network runs a {{% tts %}} Community Edition, which is a crowdsourced, open and decentralized LoRaWAN network. This network is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
```

Reviewer: So are we using Edition only for Community Edition, and not for Enterprise and Open Source?

Reviewer 2: Good one. cc @laurensslats

Reviewer: I would personally drop Edition. But I'm also very much in favor to stop changing the branding all the time.

Author: also in favor of dropping edition

<!--more-->

[Login or create an account](https://console.cloud.thethings.network/) to get started with The Things Network and start using {{% tts %}} Console.

Once you have an account, get started by following instructions for adding [Devices]({{< ref "/devices" >}}), [Gateways]({{< ref "/gateways" >}}), and [Integrations]({{< ref "/integrations" >}}).

For more information about The Things Network and LoRaWAN, see [The Things Network Documentation](https://thethingsnetwork.org/docs)

> For commercial solutions, we recommend using an SLA-backed network provided by The Things Industries. Learn more about [The Things Industries](https://thethingsindustries.com/).

Reviewer 2: ```suggestion
For commercial solutions, we strongly recommend using an SLA-backed network provided by The Things Industries. Learn more about the [enterprise offerings](https://www.thethingsindustries.com/deployment/) of [The Things Industries](https://thethingsindustries.com/).
```

Reviewer: nit; inconsistent www. vs no-www.

> For commercial solutions, we recommend using an SLA-backed network provided by The Things Industries. Learn more about [The Things Industries](https://thethingsindustries.com/).

Reviewer 2: ```suggestion
For commercial solutions, we strongly recommend using an SLA-backed network provided by The Things Industries. Learn more about the [enterprise offerings](https://www.thethingsindustries.com/deployment/) of [The Things Industries](https://thethingsindustries.com/).
```

## Clusters

> The Things Network is a multi-cluster deployment. This means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.

Reviewer 2: ```suggestion
{{% tts %}} Community Edition by The Things Network is a multi-cluster deployment. We use these clusters for different geographic regions, which means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.
```

The Things Network currently has the following clusters:

| **Cluster ID** | **Name**        | **Location**      |
| -------------- | --------------- | ---------------   |
| `au1`          | Australia 1     | Sydney, Australia |
| `eu1`          | Europe 1        | Ireland           |

> | `nam1`         | North America 1 | California, USA   |

Reviewer 2: ```suggestion
| `nam1`         | North America 1 | California, USA   |

To see how to obtain the links to these clusters, please refer to the next section.
```

---
title: "The Things Network"
description: ""
distributions: "Community"
---

{{< figure src="logo_TTN.svg" class="float plain" width="150px" >}}

The Things Network is a global collaborative Internet of Things ecosystem that creates networks, devices and solutions using LoRaWAN.

> The Things Network runs {{% tts %}} Community, which is a crowdsourced, open and decentralized LoRaWAN network. This network is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.

Reviewer 3: ```suggestion
The Things Network runs {{% tts %}} Community Edition, which is a crowdsourced, open and decentralized LoRaWAN network. This network is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
```

<!--more-->

> [Login or create an account](https://console.cloud.thethings.network/) to get started with The Things Network and start using {{% tts %}} Console.

Reviewer 3: Maybe say something like "Choose the cluster closest to you geographically, then log in or create an account to......"

Author: why?

Once you have an account, get started by following instructions for adding [Devices]({{< ref "/devices" >}}), [Gateways]({{< ref "/gateways" >}}), and [Integrations]({{< ref "/integrations" >}}).

For more information about The Things Network and LoRaWAN, see [The Things Network Documentation](https://thethingsnetwork.org/docs)

For commercial solutions, we strongly recommend using an SLA-backed network provided by The Things Industries. Learn more about the [enterprise offerings](https://thethingsindustries.com/deployment/) of [The Things Industries](https://thethingsindustries.com/).

## Clusters

> The Things Stack Community (run by The Things Network) is a multi-cluster deployment. This means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.

Reviewer 3: ```suggestion
The Things Stack Community Edition (run by The Things Network) is a multi-cluster deployment. This means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.
```

> The Things Network currently has the following clusters:

Reviewer 3: ```suggestion
{{% tts %}} Community currently has the following clusters:
```

| **Cluster ID** | **Name**        | **Location**      |
| -------------- | --------------- | ---------------   |
| `au1`          | Australia 1     | Sydney, Australia |
| `eu1`          | Europe 1        | Ireland           |
| `nam1`         | North America 1 | California, USA   |

> For links to these clusters, see the [addresses section]({{< relref "addresses" >}}).

Reviewer 3: ```suggestion
For links to these clusters, see the [Addresses section]({{< relref "addresses" >}}).
```
